app/components/files/file_info.tsx

```diff
@@ -91,13 +96,16 @@ const FileInfo = ({disabled, file, channelName, showDate, onPress}: FileInfoProp
                         <Text style={style.infoText}>
                             {`${getFormattedFileSize(file.size)}`}
                         </Text>
-                        {showDate &&
-                            <FormattedDate
-                                style={style.infoText}
-                                format={FORMAT}
-                                value={file.create_at as number}
-                            />
-                        }
+                        {showDate && (
+                            <>
+                                {' • '}
+                                <FormattedDate
+                                    style={style.infoText}
+                                    format={FORMAT}
+                                    value={file.create_at as number}
```

Comment:
Why is the casting needed? If `file.create_at` may be undefined, I think it would be safe to default to 0 (or handle such case somehow with a placeholder or not showing the component).

Reply:
The cast was already there but I surely can improve that part !